src/py42/sdk/__init__.py

```diff
@@ -44,77 +76,74 @@ def from_local_account(cls, host_address, username, password):
         Returns:
             :class:`py42.sdk.SDKClient`
         """
-        session_impl = Session
-        session_factory = SessionFactory(
-            session_impl, SessionModifierFactory(), AuthHandlerFactory()
-        )
-        basic_auth_session = session_factory.create_basic_auth_session(
-            host_address, username, password
-        )
-        sdk_dependencies = SDKDependencies(
-            host_address, session_factory, basic_auth_session
-        )
-        return cls(sdk_dependencies)
+        auth = None
+        if username and password:
```

Comment:
this is essentially what makes .netrc work, we just don't pass an `auth` object in if no username or password is supplied.